src/sandstorm/run-bundle.c++

```diff
@@ -290,7 +292,17 @@ public:
     }
   }
 
-  ~CurlRequest() {
+  explicit CurlRequest(kj::StringPtr url, int outFd): url(kj::heapString(url)) {
+    KJ_SYSCALL(pid = fork());
+    if (pid == 0) {
+      KJ_SYSCALL(dup2(outFd, STDOUT_FILENO));
+      KJ_SYSCALL(execlp("curl", "curl", isatty(STDERR_FILENO) ? "-f" : "-fs",
```

Comment:
Today I learned we rely on the host's copy of curl for updates.  I found this surprising, given that we tend to ship our own copy of basically everything else.

I guess it was just easier to reuse curl (which we already depended on at install time) than integrate an HTTP client?